thirdparty/mariadb/zimbra-mariadb/debian/rules

```diff
@@ -53,6 +53,8 @@ override_dh_auto_install:
 	rm -f $$(pwd)/debian/tmpOZCL/libmysqlclient_r.so.18
 	rm -f $$(pwd)/debian/tmpOZCL/libmysqlclient_r.so.18.0.0
 	cd $$(pwd)/debian/tmpOZCL && \
+	ln -s libmariadb.so.3 libmysqlclient.so.18 && \
```

Comment:
Please include some evidence of the issue.  The commit comment describes the fix not why it was done.  What breaks if this is not done?

As an aside, it seems odd to have different versions on the shared libraries (but clearly if it is necessary, then fine)